dev-docs/releasing.adoc

```diff
@@ -0,0 +1,73 @@
+
+=== Where to find the release wizard
+
+The release wizard is found in a checkout of solr at `dev-tools/scripts/releaseWizard.py`
+
+=== What working copy to use
+
+The release wizard is meant to be used from any working copy you like, with the expectation that you will check out the PARENT branch. In other words you should check out main to publish M.0.0 and branch_Mx to publish M.N.0, and branch M.N.x-1 for M.N.x. The wizard will (eventually) guide you into creating a fresh, clean checkout where your intended release is built, but you don't need that at the start.
+
+=== How to run the release wizard
+
+1. Make sure you have python 3.4+ installed (if a higher version becomes required, the wizard should complain and tell you what it is)
+2. Install dependencies with `pip3 install -r requirements.txt`, from the `dev-tools/scripts` folder.
+3. Run the command you see documented at `dev-tools/scripts/README.md`. Using `--dry-run` initially is fine.
+
+NOTE: `--dry-run` does still create `~/.solr_releases` and `~/.solrrc` files and record the release version you intend for future reference so one might say it's really better described as a "slightly damp" run :) it should however not execute other commands. (in theory). Also, checklist elements approved are also remembered, but this may change, see https://issues.apache.org/jira/browse/SOLR-17246[SOLR-17246]
+
+
+== Step 2 - Complete the First two Checklists
+
+The release wizard is organized into checklists and the first two checklists are preparation/planning related, and are good to complete before proposing the release.
+
+== Step 3 - Propose the Release
+
+Mail dev@solr.apache.org with a subject like `[DISCUSS] Solr X.Y release` and wax poetic about the wonderful features and awful bugs that are fixed but not yet released and volunteer to do the release. If you are sufficiently inspiring most of the PMC will quietly say to themselves "Whew! I don't have to do it" and then some of them will respond with a +1. Then at the last minute right before the time you proposed to start the release 1-4 people will suddenly mail the list saying something like "Can we wait for X amount of time so that I can get SOLR-XXXXXX in?" Let the community discuss it, and give them some time because you knew this would happen (because you read this guide!)
+
+== Step 4 - Continue Following the Release Wizard
+
+It's supposed to tell you all you need from here... Mail the list if you have questions, and file Jira issues for things that could be improved for the next person (please set component to `release-scripts`).
+
+== Step 5 - Complete the After Release Steps in the Wizard too.
+
+There are some things that need to be done after the release is officially published and announced. The wizard will guide you through that (things like updating Jira etc.)
+
+
```

Comment:
== Dive deeper

The [Release Creation Process](https://infra.apache.org/release-publishing.html) we page gives more in-depth explanation of an Apache release.

The origins of the ReleaseWizard is described in [this blog post](https://www.linkedin.com/pulse/releasing-lucene-just-61-steps-jan-h%C3%B8ydahl/) from 2019. The same tool is currently used by Lucene, Solr and Solr-Operator.